### PR TITLE
git-sed: add page

### PR DESCRIPTION
**Pull request by deadpyxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 6.2.0 (Aur package)

Opened from #5137

### CODE REVIEW

File: pages/common/git-sed.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# git sync
+
+> Run grep as directed but replace the given files with the pattern.
```
Comment (navarroaxel) [inline pages/common/git-sed.md:3]: This description says nothing to me.

File: pages/common/git-sed.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+- Replace the string 'this' with 'that':
+
+`git sed 'this' 'that'`
+
+- Replace the string and then commit the resulting changes with a standard commit message:
+
+`git sed -c 'this' 'that'`
+
+- Replace the string passing regex flags to `sed` :
```
Comment (navarroaxel) [inline pages/common/git-sed.md:16]:
```suggestion
- Replace the string passing regex flags to `sed`:
```

File: pages/common/git-sed.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+`git sed -f g 'this' 'that'`
+
+- Replace the string only in the given pathspec:
+
+`git sed 'this' 'that' -- lake/`
```
Comment (navarroaxel) [inline pages/common/git-sed.md:22]:
```suggestion
`git sed '{{find_text}}' '{{replace_text}}' -- {{path/to/directory}}`
```

File: pages/common/git-sed.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+`git sed 'this' 'that'`
+
+- Replace the string and then commit the resulting changes with a standard commit message:
+
+`git sed -c 'this' 'that'`
+
+- Replace the string passing regex flags to `sed` :
+
+`git sed -f g 'this' 'that'`
```
Comment (navarroaxel) [inline pages/common/git-sed.md:18]: The docs doesn't use `-f` to use the `g`

File: pages/common/git-sed.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+- Replace the string only in the given pathspec:
```
Comment (navarroaxel) [inline pages/common/git-sed.md:20]:
```suggestion
- Replace a specific text from files under a given directory:
```

File: pages/common/git-sed.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# git sync
+
+> Run grep as directed but replace the given files with the pattern.
+> Works only on git tracked files.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
+
+`git sed 'this' 'that'`
```
Comment (navarroaxel) [inline pages/common/git-sed.md:10]: What if we use the placeholders from the `sed` page
```suggestion
`git sed '{{find_text}}' '{{replace_text}}'`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages/common/git-sed.md (hunk 4 above)
Comment (deadpyxel) [reply pages/common/git-sed.md:18]: Yes, I noticed it. I was looking at the man page and there's mention of using `-f g`

Not sure if I should follow the man page or the docs
image

File: pages/common/git-sed.md (hunk 6 above)
Comment (deadpyxel) [reply pages/common/git-sed.md:10]: Makes sense, thanks for the suggestion.

**Comment by deadpyxel** [comment]
Thanks for the suggestions @navarroaxel I'll be working on your suggestions

File: pages/common/git-sed.md (hunk 1 above)
Comment (deadpyxel) [reply pages/common/git-sed.md]: I have updated the description on 226a057 please let me know if that is better.

File: pages/common/git-sed.md (hunk 4 above)
Comment (marchersimon) [reply pages/common/git-sed.md]: I think as long as it works it's not important.

File: pages/common/git-sed.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+# git sync
+
+> Replace patterns in git-controlled files using sed.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-sed>.
+
+- Replace the string 'this' with 'that':
```
Comment (marchersimon) [inline pages/common/git-sed.md:7]: In all files in the repository? Or in the current repository?

File: pages/common/git-sed.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+`git sed '{{find_text}}' '{{replace_text}}'`
+
+- Replace the specified text and then commit the resulting changes with a standard commit message:
+
+`git sed -c '{{find_text}}' '{{replace_text}}'`
+
+- Replace the specified text passing regex flags to `sed`:
```
Comment (marchersimon) [inline pages/common/git-sed.md:15]: How about this?
```suggestion
- Replace the specified text, using regular expressions:
```

File: pages/common/git-sed.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+`git sed -f g '{{find_text}}' '{{replace_text}}'`
+
+- Replace a specific text from files under a given directory:
```
Comment (marchersimon) [inline pages/common/git-sed.md:19]:
```suggestion
- Replace a specific text in all files under a given directory:
```

File: pages/common/git-sed.md (hunk 8 above)
Comment (deadpyxel) [reply pages/common/git-sed.md:15]: That sounds good to me. Thanks for the suggestion.

File: pages/common/git-sed.md (hunk 7 above)
Comment (navarroaxel) [reply pages/common/git-sed.md:7]: Perhaps this better
```suggestion
- Replace the specified text in the current repository:
```

File: pages/common/git-sed.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
+# git sync
```
Comment (marchersimon) [inline pages/common/git-sed.md:1]:
```suggestion
# git sed
```

Comment (navarroaxel) [reply pages/common/git-sed.md:1]: I think we can apply this small fix then merge this.

Comment (deadpyxel) [reply pages/common/git-sed.md]: That completely slipped me, I'm sorry. Thanks for pointing it out.

**Review by marchersimon — APPROVED** [review]
Thanks @deadpyxel

**Pull request merged**